CONTRIBUTING.md

```diff
@@ -37,6 +38,19 @@ The easier it is for us to reproduce it, the faster it'll be fixed!
 
 Please don't include any private/sensitive information in your issue!
 
+## Providing Extensions
+
+Some of the best features of Podman Desktop aren't even in this repository!
+We've created an API so that the capabilities of Podman Desktop can be expanded
```

Comment:
question: why we ? Rather than Podman Desktop

And why is in the past ? Created and extended ? 
Example: Podman Desktop provides a set of API expanding its capabilities

Reply:
Was trying to be (a bit too) informal. Fixed in new commit.